firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -42,6 +42,7 @@ export interface FirestoreBigQueryEventHistoryTrackerConfig {
   tableId: string;
   datasetLocation: string | undefined;
   tablePartitioning: string;
+  clustering: string[] | null;
```

Comment:
Shouldn't this be:

```ts
  clustering?: string[];
```

We're incrementing the version as minor, but this change in it's current state will require a value making it a breaking change (since users existing code won't include that value).

Reply:
Should #852 be changed in the config nad index file where I supply it to be undefined then or conditionally attached somehow then afterwards? I thought that until we won't release #852 with updated version to 1.1.14 on package.json the users will be still using the firestore-extension-change-tracker package in ver 1.1.13 before #852 will be released.